client/consensus/Cargo.toml

```diff
@@ -39,10 +39,13 @@ cumulus-client-consensus-proposer = { workspace = true }
 cumulus-primitives-core = { workspace = true }
 cumulus-primitives-parachain-inherent = { workspace = true }
 cumulus-relay-chain-interface = { workspace = true }
+cumulus-test-relay-sproof-builder = { workspace = true }
 
 # Polkadot
+polkadot-core-primitives = { workspace = true }
 polkadot-node-primitives = { workspace = true }
 polkadot-overseer = { workspace = true }
+polkadot-parachain-primitives = { workspace = true }
```

Comment:
New dependencies that are only used in tests should be under `[dev-dependencies]`.